gobblin-utility/src/main/java/gobblin/util/ExecutorsUtils.java

```diff
@@ -28,6 +30,7 @@
  *
  * @author ynli
  */
+@Slf4j
```

Comment:
Can we avoid using lombok for this? The savings it brings (one line of code) does not justify the amount of confusion it brings in at the same time. I prefer defining a constant `LOGGER` explicitly.